plugin.js

```diff
@@ -160,7 +160,10 @@ const buildAuthenticatedRouter = (admin, auth, predefinedRouter, sessionOptions
     const adminUser = await auth.authenticate(email, password)
     if (adminUser) {
       req.session.adminUser = adminUser
-      res.redirect(rootPath)
+      req.session.save((err) => {
+        console.error(err)
```

Comment:
I would move that error to `next` middleware instead of console logging that:

```
if (error) {
  next(error)
}
```
where next is the third argument of router.post method.

I made this mistake in previous expressjs and Sentry integration didn't work.

Reply:
fixed